src/main.ts

```diff
@@ -31,7 +31,10 @@ export const vibClient = clients.newClient({
       : constants.DEFAULT_VIB_PUBLIC_URL
   }`,
   timeout: 10000,
-  headers: { "Content-Type": "application/json", "User-Agent": "VIB/0.1" },
+  headers: {
+    "Content-Type": "application/json",
+    "User-Agent": "vib-action/0.1-dev",
```

Comment:
We should have a parameter for the version. If not, when we cut a new release, we could forget to update this string.

Reply:
Then something like this? :

Reply:
Something like this? 

`"User-Agent": "vib-action"
"Version":"0.1-dev"`

Reply:
Nah. What @recena  is suggesting is to remove the 0.1-dev altogether and pick that information dynamically. 

I've made a quick test in the old repository creating a release ( 0.1.0-dev tag ) and then printing the env vars and it seems that the version number of the GitHub Action will come as an environment variable:

<img width="737" alt="Screen Shot 2022-02-22 at 11 39 39" src="https://user-images.githubusercontent.com/584642/155115907-5a02e672-a055-45bc-baf3-e7821cf20167.png">

So @gdelgadot essentially you need to change that 0.1-dev with whatever comes in the environment variable or some default (e.g. "unknown") if nothing is found.